src/templates/static/mt5_signals.jsx

```diff
@@ -0,0 +1,240 @@
+                                                        <BoxInner className='border-right-top' image='automated_copy_trade' text={it.L('Automated copying – no extra work from you')} />
+                                                    </Box>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div className='gr-padding-30' >
+                                <div className='container gr-padding-30 gr-child'>
+                                    <h2 className='center-text'>
+                                        {it.L('How to register as a signal provider')}
+                                    </h2>
+                                    <div className='gr-row gr-padding-30'>
+                                        <div className='gr-4 gr-12-m gr-padding-10 gr-child'>
+                                            <img className='responsive' src={it.url_for('images/pages/mt5_signals/signal_provider_register_1.png')} />
+                                            <ol>
+                                                <li>{it.L('Go to the MQL5 signals showcase page and click on the <strong>Create signal</strong> button.')}</li>
+                                            </ol>
+                                        </div>
+                                        <div className='gr-4 gr-12-m gr-padding-10 gr-child'>
+                                            <img className='responsive' src={it.url_for('images/pages/mt5_signals/signal_provider_register_2.png')} />
+                                            <ol start='2'>
+                                                <li>{it.L('Complete the form with your Deriv MT5 account credentials. For the <strong>Broker</strong> field, add <strong>Deriv-Server</strong> as your broker or <strong>Deriv-Demo</strong> if your signal is for demo accounts only.')}</li>
+                                            </ol>
+                                        </div>
+                                        <div className='gr-4 gr-12-m gr-padding-10 gr-child'>
+                                            <img className='responsive' src={it.url_for('images/pages/mt5_signals/signal_provider_register_3.png')} />
+                                            <ol start='3'>
+                                                <li>{it.L('Click <strong>OK</strong> and add a description to complete the registration')}</li>
+                                            </ol>
+                                        </div>
+                                    </div>
+                                    <div className='gr-padding-30' >
+                                        <p>{it.L(' <strong>Note:</strong> ')}</p>
```

Comment:
here, the space and strong can just be outside of the translation string completely. I'm not sure why space is needed though.